src/main.js

```diff
@@ -0,0 +1,91 @@
+const classes = {
+  AFTERNOON: 'afternoon',
+  MORNING: 'morning',
+  EVENING: 'evening',
+};
+
+const selectors = {
+  TIME: '.time',
+  GREETING: '.greeting',
+  NAME: '.name',
+  FOCUS: '.focus',
+};
+
+const nameElement = document.querySelector(selectors.NAME);
+const focusElement = document.querySelector(selectors.FOCUS);
+
+function addZeroFirstFormat(value){
+  return value < 10 ? '0' + value : value;
+}
+
+function showTime() {
+  const timeElement = document.querySelector(selectors.TIME);
+  let currentTime = new Date();
+  let hour = currentTime.getHours();
+  let min = addZeroFirstFormat(currentTime.getMinutes());
+  let sec = addZeroFirstFormat(currentTime.getSeconds());
+
+  timeElement.innerHTML = `${hour}:${min}:${sec}`;
+  setTimeout(showTime, 1000);
+}
+
+function setGreetingLine() {
+  const greetingElement = document.querySelector(selectors.GREETING);
+  let currentTime = new Date();
+  let hour = currentTime.getHours();
+
+  if (hour < 12) {
+    greetingElement.textContent = 'Good Morning, ';
+    document.body.classList.add(classes.MORNING);    
+  } else if (hour < 18) {
+    greetingElement.textContent = 'Good Afternoon, ';
+    document.body.classList.add(classes.AFTERNOON); 
+  } else {
+    greetingElement.textContent = 'Good Evening, ';
+    document.body.classList.add(classes.EVENING);
+  }
+}
+
+function getName() {
```

Comment:
![image](https://user-images.githubusercontent.com/25482018/106502055-ee5f4f00-64d4-11eb-87d8-f6387b4c79f4.png)
or it shill can be getName, but you need to change a bit inner logic and add return